cms/publications/templates/publications/publication.html

```diff
@@ -70,8 +71,41 @@
         </div>
         <hr>
         <h1>{{ self.title }}</h1>
-        <div id="content">{{ self.body|richtext }}</div>
         {% if self.publication_publication_type_relationship %}
+        <div class="nhsuk-grid-row">
+            <div class="nhsuk-grid-column-one-third nhsuk-u-margin-top-4 content_nav">
+                <nav>
+                    {% get_toc page %}
+                    <a href="#related_contents">Related Content</a>
+                </nav>
+            </div>
+            <div class="nhsuk-grid-column-two-thirds nhsuk-u-margin-top-4">
+                <div id="content">{{ self.body|richtext }}</div>
+                <hr>
+                <div id="see-all-updates" class="nhsuk-review-date last-publication">
+                    <p class="nhsei-body-s">
+                        <strong>Date published:</strong> {{ self.first_published_at|date:'d M Y' }}<br>
+                        <strong>Date Last updated:</strong> {{ self.latest_revision_created_at|date:'d M Y' }}
```

Comment:
Bikesheddy: I feel like the first letters of 'last' and 'updated' should both be capitalised, or neither.

Reply:
I think the style manual says sentence case is right here, "Date published" and "Date last updated".